packages/lib-react-components/src/PlainButton/README.md

```diff
@@ -11,7 +11,10 @@ Example use cases:
 
 ```
 <PlainButton
-  color={'teal'}
+  color={
+    dark: "#2d2d2d",
+    light: "#eff2f5"
+  }
```

Comment:
Almost there! Grommet expects this to be an object prop _passed into_ the brackets. Because of that this should actually read:

```
color={{
    dark: "#2d2d2d",
    light: "#eff2f5"
}}
```

Sorry I could've made that a bit clearer in previous comment.

Reply:
Good catch! Fixed.